robottelo/common/constants.py

```diff
@@ -56,6 +61,7 @@
 #       filter's common_locator['filter']
 FILTER = {'arch_os': "architecture_operatingsystem",
```

Comment:
Sort the entries, end all lines with , and open and close {} on its own line. Use single quotes instead of double quotes.